workshops/models.py

```diff
@@ -329,6 +336,34 @@ def __str__(self):
     def get_absolute_url(self):
         return reverse('event_details', args=[self.get_ident()])
 
+    def get_repository_url(self):
+        """Return self.url formatted as it was repository URL.
+
+        Repository URL is: https://github.com/user/repository/
+        If it doesn't match, the original URL is returned."""
+        try:
+            mo = self.WEBSITE_REGEX.match(self.url)
```

Comment:
I would not use an exception here, since it's not clear that you're relying on `mo` being `None`. Instead, I would use:

~~~
mo = ...match...
if mo:
    return blah
else:
    return blarg
~~~